docs/tutorials/register.md

```diff
@@ -111,7 +111,7 @@ print(f"{reg.nodes = }") # markdown-exec: hide
 print(f"{reg.edges = }") # markdown-exec: hide
 ```
 
-It is possible to customize qubit interaction through the [`add_interaction`][qadence.transpile.emulate.add_interaction] method.
```

Comment:
FYI there are a few places where I simply removed the reference to the API. For some weird reason setting it as `qadence.analog.add_interaction.add_interaction` never worked, and either way in the next reworks I would like to hide this function from the user.